src/lib/icons/IconThumbDown.svelte

```diff
@@ -1,12 +1,28 @@
-<!-- source: https://fonts.google.com/icons?selected=Material%20Icons%3Athumb_down%3A -->
+<!-- source: DFINITY foundation -->
 <svg
+  width="48"
+  height="48"
+  viewBox="0 0 48 48"
+  role="img"
+  aria-label="Dislike or Thumbs Down Icon"
```

Comment:
We can go with `role` + `aria-label` or hide them completely with `aria-hidden="true"`. Any preferences?

Reply:
This looks like a nice improvement; however, I’m also not sure if this is the right place to add text in the components library. These icons could also be used for something else, and generally, this type of label comes from the en.json file in the nns-dapp.